Makefile

```diff
@@ -90,6 +90,8 @@ ifeq ("$(TARGETOS)", "trusty")
 	TARGETGOARCH := $(HOSTARCH)
 endif
 
+SYZ_CLANG ?= 0
```

Comment:
SYZ_CLANG needs to be passed to tools/syz-env above, syz-env is what figures how to build native binaries for the makefile.